modules/tinymce/src/core/test/ts/browser/textpatterns/TextPatternsTest.ts

```diff
@@ -213,4 +213,17 @@ describe('browser.tinymce.core.textpatterns.TextPatternsTest', () => {
     TinyContentActions.keyup(editor, Keys.space());
     TinyAssertions.assertContent(editor, '<p><a href="about:blank"><span class="test">www</span>.google.com</a></p>');
   });
+
+  it('TINY-8540: should be able to add additional patterns (connections requirement)', () => {
```

Comment:
Could you unset the option as well so that the default gets set back if someone adds a test after this.

- [ ] Unset option so that the default sets restored for other tests

Reply:
Also not sure we need to mention that this is for a particular product that is an internal concern and can be tracked in JIRA? We don't generally do this for other things if it's customer specific fixes etc.

Reply:
Yeah fair point, I was mainly trying to make it very clear why this exists in case someone looks at it in a couple years as they're less likely to track it back then.

Also good catch on resetting the option value.

Reply:
Have removed the reference and added the `unset` call at the end.